src/ucs/arch/bitops.h

```diff
@@ -94,15 +94,17 @@
 )
 
 /* Returns the number of 1-bits in x */
-#define ucs_count_one_bits(x)      __builtin_popcount(x)
+#define ucs_popcount(x) \
```

Comment:
minor: would need to use `_` prefix in macro names

Reply:
yeah, i didn't want to fix this issue too in this PR..

Reply:
@brminich fixed